smartsim/_core/control/controller.py

```diff
@@ -385,6 +385,7 @@ def symlink_output_files(
                 "Symlinking files failed."
             ) from fnf
 
+    # pylint: disable-msg=too-many-locals
```

Comment:
I know you didn't create the problem but it does look like we could probably break out a few chunks of this method into new methods and avoid the warning altogether. Can you give this a once over to see if anything is easy to pull out before we commit to disabling `too-many-locals`?

Maybe we could pull out the `for orchestrator in manifest.dbs:` loop into `_launch_orchestrators`, for example...

Reply:
`max-locals` got bumped to 25, so this is no longer an issue and I'll take out the comment!